src/index.tsx

```
@@ -137,13 +155,101 @@ export class Terminal extends Component<TerminalProps, TerminalState> {
       }
     };
 
+    const handleTabPress = async (): Promise<void> => {
+      const {
+        autoCompleteActiveItem,
+        autoCompleteIsActive,
+        autoCompleteItems,
+        inputValue,
+        currentPath,
+        fileSystem,
+      } = this.state;
+      const { commandName, commandTargets } = parseCommand(inputValue);
+
+      let newAutoCompleteActiveItem = 0;
+      if (autoCompleteIsActive && autoCompleteItems) {
+        const previewItemCount = Object.keys(autoCompleteItems).length;
+
+        if (autoCompleteActiveItem < previewItemCount - 1) {
+          newAutoCompleteActiveItem = autoCompleteActiveItem + 1;
+        }
+
+        let newInputValue = `${inputValue}${
+          Object.keys(autoCompleteItems)[newAutoCompleteActiveItem]
+        }`;
+
+        if (commandTargets[0].length > 0) {
+          newInputValue = inputValue.replace(
+            commandTargets[0],
```

Do this in a better way. Maybe throw an error if there are multiple targets.